server/datastore/mysql/software.go

```diff
@@ -2669,7 +2669,10 @@ SELECT
 FROM software_titles st
 INNER JOIN software_installers si ON si.title_id = st.id
 INNER JOIN host_software_installs hsi ON hsi.host_id = ? AND hsi.software_installer_id = si.id
-WHERE hsi.removed = 0
+WHERE hsi.removed = 0 AND 
+	-- :software_status_installed
+	((hsi.post_install_script_exit_code IS NOT NULL AND hsi.post_install_script_exit_code = 0) OR
+	(hsi.install_script_exit_code IS NOT NULL AND hsi.install_script_exit_code = 0))
```

Comment:
Should we do the same for VPP apps? (by joining with `nano_command_results` and checking the status)

Reply:
Yes, wouldn't hurt.